### DISCUSSION TITLE
expo-video not able to play video from m3u8 on ios 16

### DISCUSSION THREAD

**Discussion by TithSambath** [expo/expo#33635] [Archived: Questions]
Hello, Does anyone face the same issue as mine on expo 51, regarding to expo-video not able to play video from m3u8 only on ios 16.

**Comment by mkhbragg** [comment]
I'm using Expo SDK 52 and my `.m3u8` video works fine on iPhone 15 Pro when using `expo-video`, but if I want to stream the same video on iPhone 16 Pro it doesn't work with `expo-video`. The `.m3u8` video does work on iPhone 16 Pro if I use `expo-av` instead of `expo-video`.

**Comment by arapocket** [comment]
It might be your iOS version and not your iphone that's the problem. Airplay stopped working on all of my devices that are on iOS 18.4.1

For this library and react-native-video
